.github/workflows/release-dockers.yaml

```diff
@@ -0,0 +1,28 @@
+name: Bump version and release dockers
+on:
+  push:
+    branches: main
+
+jobs:
+  bump-version:
+    uses: openclimatefix/.github/.github/workflows/bump-version.yml@main
+
+  release-inference-docker:
+    needs: bump-version
+    secrets:
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+    uses: FIXME: MAKE RELEASE-ONLY ACTION
```

Comment:
We could easily make a workflow in `openclimatefix/.github` that does that.

Reply:
What that just does a bump version?

Reply:
or just does a release?

Reply:
The way I see it we want
1. bump the version of the whole repo
2. release docker A
3. release docker B
(2 and 3 can be done in parallel but 1 has to be done first)

Would that work?